Rakefile

```diff
@@ -65,9 +65,7 @@ task :generate_parser_caches => [PARSER_CACHE_DIR] do
 
     cache = YAML.load_file(filename.to_s)
     cache[:files] = cache[:files].delete_if do |k, _|
-      k =~ /\Aforeman/ &&
-        k !~ /\Aforeman_proxy_content/ &&
-        k !~ /\Aforeman_proxy_certs/
+      !%w(certs foreman_proxy_content katello puppet).include?(k)
```

Comment:
Agreed, based on the initial comment I assumed puppet would be omitted here, but I guess thats the point. The current error revolves around there not being a puppet parser cache found:

```
stderr: /usr/share/gems/gems/kafo-1.0.6/lib/kafo/puppet_module.rb:69:in `parse': No Puppet module parser is installed and no cache of the file /usr/share/foreman-installer/modules/puppet/manifests/init.pp is available. Please check debug logs and install optional dependencies for the parser. (Kafo::ParserError)
        from /usr/share/gems/gems/kafo-1.0.6/lib/kafo/configuration.rb:93:in `block in modules'
        from /usr/share/gems/gems/kafo-1.0.6/lib/kafo/configuration.rb:93:in `map'
        from /usr/share/gems/gems/kafo-1.0.6/lib/kafo/configuration.rb:93:in `modules'
        from /usr/share/gems/gems/kafo-1.0.6/lib/kafo/configuration.rb:198:in `params'
        from /usr/share/gems/gems/kafo-1.0.6/lib/kafo/configuration.rb:208:in `preset_defaults_from_puppet'
        from /usr/share/gems/gems/kafo-1.0.6/lib/kafo/kafo_configure.rb:277:in `set_parameters'
        from /usr/share/gems/gems/kafo-1.0.6/lib/kafo/kafo_configure.rb:100:in `initialize'
        from /usr/share/gems/gems/clamp-1.0.0/lib/clamp/command.rb:133:in `new'
        from /usr/share/gems/gems/clamp-1.0.0/lib/clamp/command.rb:133:in `run'
        from /usr/share/gems/gems/kafo-1.0.6/lib/kafo/kafo_configure.rb:155:in `run'
        from /sbin/foreman-installer:8:in `<main>'
        to retry, use: --limit @/home/ehelms/.ansible/retry-files/nightly_server_proxy_client.retry
```